apps/hub-app/src/pages/HomePage.tsx

```diff
@@ -159,6 +168,8 @@ const HomePage = () => {
           toggleDelegateFilter={toggleDelegateFilter}
           sortBy={sortBy}
           toggleSortBy={toggleSortBy}
+          searchTerm={searchTerm}
```

Comment:
My bad on all this prop threading. 

We can restructure this once we reincorporate the layout elements.